plugins/wepselect.lua

```diff
@@ -124,27 +124,37 @@ if (CLIENT) then
 	function PLUGIN:PlayerBindPress(client, bind, pressed)
 		bind = bind:lower()
 
-		if (!pressed or !bind:find("invprev") and !bind:find("invnext")
+		local bScrollUp = bind:find("invprev")
+		local bScrollDown = bind:find("invnext")
+
+		if (!pressed or !bScrollUp and !bScrollDown
 		and !bind:find("slot") and !bind:find("attack")) then
 			return
 		end
 
-		local currentWeapon = client:GetActiveWeapon()
-		local bValid = IsValid(currentWeapon)
-		local bTool
-
-		if (client:InVehicle() or (bValid and currentWeapon:GetClass() == "weapon_physgun" and client:KeyDown(IN_ATTACK))) then
+		if (client:InVehicle()) then
 			return
 		end
 
-		if (bValid and currentWeapon:GetClass() == "gmod_tool") then
-			local tool = client:GetTool()
-			bTool = tool and (tool.Scroll != nil)
+		local currentWeapon = client:GetActiveWeapon()
+		local bValid = currentWeapon:IsValid()
+
+		if (bValid and currentWeapon:GetClass() == "weapon_physgun" and client:KeyDown(IN_ATTACK)) then
+			return
 		end
 
 		local weapons = client:GetWeapons()
+		if #weapons == 0 then return end
```

Comment:
format to consistent style with helix

https://cs.github.com/NebulousCloud/helix/blob/4d1922407c93828cfd951aec70f89e834ee0f836/gamemode/core/libs/cl_hud.lua#L7
https://cs.github.com/NebulousCloud/helix/blob/4d1922407c93828cfd951aec70f89e834ee0f836/gamemode/core/libs/sh_storage.lua#L314